### PR TITLE
General "getting started" information

### PR DESCRIPTION
**Pull request by daggerhart**
Some very general overview of requirements, installation, and examples.

I'm not sure if you'd recommend installing this project globally. Happy to change if there is a better approach.

### CODE REVIEW

**Comment by mglaman** [comment]
Yeah this should be a global command. Will review, thanks!

File: README.md [hunk 1]
```diff
@@ -34,3 +34,27 @@ Available commands:
   travisci:list (tci:l)        Lists Travis Ci builds for a Drupal project
   travisci:watch (tci:w)       Watches a Travis CI job
 ````
+
+## Getting Started
+
+### Requirements
+
+* [git](https://git-scm.com/)
```
Comment (mglaman) [inline README.md]: Should we explain this is only needed if contributing? Also I wonder if there should be a `patch` command fallback if the repository is not in Git

File: README.md [hunk 2]
```diff
@@ -34,3 +34,27 @@ Available commands:
...
+* [composer](https://getcomposer.org/)
```
Comment (mglaman) [inline README.md]: Let's just specify that this is just for for installing the tool

File: README.md [hunk 3]
```diff
@@ -34,3 +34,27 @@ Available commands:
...
+
+### Installation
+
+After installing git and composer:
```
Comment (mglaman) [inline README.md]: Let's simplify this to just "Use the following command to install the command line tool via Composer "

File: README.md [hunk 4]
```diff
@@ -34,3 +34,27 @@ Available commands:
...
+
+`composer global require mglaman/drupalorg-cli`
+
+### Use
+
+Since this tool uses git for a significant amount of its operations, we must first clone a project from drupal.org:
```
Comment (mglaman) [inline README.md]: Hrm. So really only `issue:branch` and `issue:apply` use git.

The tool provides a lot of other helpers, like CI watching and stuff.

**Comment by daggerhart** [comment]
I've updated the PR with the changes you requested (according to my understanding). I have a little more time today to make additional changes, and I can smash these changes into a single commit if you'd like.

**Comment by mglaman** [comment]
Thanks! Will review when off phone. No need to squash

**Comment by mglaman** [comment]
Thanks @daggerhart !

**Pull request merged**